test/rules/no-unbound-method/test.ts.lint

```diff
@@ -25,4 +25,8 @@ i.foo;
 ~~~~~ [0]
 i.bar;
 
+
+<button onClick={c.method}>Click me!</button>;
```

Comment:
Excuse my curiosity, but wouldn't this need to be a `.tsx.lint` file for this to be parsed correctly?